### PR TITLE
Show command review

### PR DESCRIPTION
**Pull request by jiaweiteo**
Increased test coverage for show command to 90%

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ShowCommandParser.java [hunk 1]
```diff
@@ -14,16 +14,13 @@
...
+     * @throws ParseException if the user input does not conform to the expected format.
      */
     public ShowCommand parse(String args) throws ParseException {
-        String trimmedArgs = args.trim();
-        if (trimmedArgs.isEmpty()) {
-            throw new ParseException(
-                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, ShowCommand.MESSAGE_USAGE));
-        }
         String[] keywords = args.split(" ");
+        if (keywords.length != ShowCommand.COMMAND_TOKENS) {
```
Comment (lyeyixian) [inline src/main/java/seedu/address/logic/parser/ShowCommandParser.java:21]: Nice way to work around the magic numbers! @timjkong @underthehai

File: src/test/java/seedu/address/logic/command/ShowCommandTest.java [hunk 2]
```diff
@@ -1,14 +1,41 @@
...
+import seedu.address.logic.command.exceptions.CommandException;
 import seedu.address.logic.parser.ShowCommandParser;
 import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
 
 public class ShowCommandTest {
+
+    private Model model;
```
Comment (lyeyixian) [inline src/test/java/seedu/address/logic/command/ShowCommandTest.java:24]: Maybe consider using a stub?

File: src/test/java/seedu/address/logic/command/ShowCommandTest.java [hunk 3]
```diff
@@ -18,12 +45,112 @@ public void execute_show_success() {
     }
 
     @Test
-    public void execute_goto_failure() {
+    public void execute_showAccommodation_success() {
+        try {
```
Comment (lyeyixian) [inline src/test/java/seedu/address/logic/command/ShowCommandTest.java:49]: I wonder why you didn't use the method in CommandTestUtil (assertCommandSuccess(), etc.)?

File: src/test/java/seedu/address/logic/command/ShowCommandTest.java (hunk 2 above)
Comment (lyeyixian) [reply src/test/java/seedu/address/logic/command/ShowCommandTest.java:24]: Or is it suppose to use original model?

**Review by lyeyixian — APPROVED** [review]
LGTM. Just some nits.

File: src/test/java/seedu/address/logic/command/ShowCommandTest.java (hunk 2 above)
Comment (jiaweiteo) [reply src/test/java/seedu/address/logic/command/ShowCommandTest.java:24]: Try out integration testing? haha

**Pull request merged**